app/mailers/request_form_mailer.rb

```diff
@@ -7,7 +7,7 @@ def bpsc_read_on_site_request(bpsc_read_on_site_request)
 
   def rcrf_read_on_site_request(rcrf_read_on_site_request)
     @rcrf_read_on_site_request = rcrf_read_on_site_request
-    mail(to: ['depository@ualberta.ca', 'ill@ualberta.ca'],
+    mail(to: ['depository@ualberta.ca'],
```

Comment:
Can remove the array now since its just one person:
```suggestion
    mail(to: 'depository@ualberta.ca',
```